### ISSUE TITLE
Question about the ImageNet-100

### ISSUE THREAD

**Issue by ElonJustin7** [deeplearning-wisc/cider#12]
Hi author,
Thanks for your excellent work！I have followed the scripts and code you provided, However, I have a question about the performance of ImageNet-100. 
In your paper, you reported an average AUROC of nearly 98-99 for ImageNet-100, while my attempts to reproduce the results are yielding an average of 94. I am wondering if there might be any parameter or dataset-related discrepancies(random selected 100 classes from ImageNet-1k) leading to this variation.
 
The parameters I used are as follows:
    --model resnet34 \
    --loss cider \
    --epochs 10 \
    --proto_m 0.95 \
    --feat_dim 128 \
    --w 2 \
    --learning_rate 0.01

If possible, could you please provide me with the fine-tuned checkpoint? 
Thank you very much！
